### PR TITLE
Remove unused endpoints after PA-App & AW-App removal

### PR DESCRIPTION
**Pull request by aberonni**
[AB#27474](https://dev.azure.com/redcrossnl/7d17f2a9-3f18-40f1-8c29-98568fbca7e4/_workitems/edit/27474)

- [x] Analyse used/unused endpoints
- [x] Get approval on endpoints marked for removal
- [x] Remove unused endpoints and related dead code

As a result of the first two steps, these are the endpoint calls marked for deletion (only used in PA-App or AW-App):

```js
- [x] POST    `/notifications/lookup`
- [x] POST    `/people-affected/data-storage`
- [x] GET     `/people-affected/data-storage/${type}`
- [x] POST    `/programs/${programId}/registrations` // removed in a8737ab6dbda96f452cc15ab31d6cada53825a1d
- [x] POST    `/programs/${programId}/registrations/custom-data` // removed in a8737ab6dbda96f452cc15ab31d6cada53825a1d
- [x] POST    `/programs/${programId}/registrations/fsp`
- [x] GET     `/programs/${programId}/registrations/fsp-attributes/`
- [x] POST    `/programs/${programId}/registrations/issue-validation`
- [x] POST    `/programs/${programId}/registrations/phone` // not found - probably already removed
- [x] POST    `/programs/${programId}/registrations/register` // not found - probably already removed?
- [x] GET     `/programs/${programId}/registrations/status/${referenceId}`
- [x] GET     `/programs/published/all`
- [x] GET     `/registrations/${referenceId}`
- [x] GET     `/registrations/download/validation-data`
- [x] DELETE  `/users`
- [x] POST    `/users/person-affected` // not found - probably already removed
// Not used by anyone at all:
- [x] PATCH   `/instance/monitoringQuestion`
```

### CODE REVIEW

File: services/121-service/src/instance/instance.controller.ts [hunk 1]
```diff
@@ -49,20 +47,4 @@ export class InstanceController {
...
-
-  @AuthenticatedUser({ isAdmin: true })
-  @ApiOperation({ summary: 'Update instance monitoring question' })
-  @ApiResponse({
-    status: HttpStatus.NOT_FOUND,
-    description: 'No instance found',
-    type: InstanceEntity,
-  })
-  @Patch('monitoringQuestion')
-  public async updateMonitoringQuestion(
```
Comment (elwinschmitz) [inline services/121-service/src/instance/instance.controller.ts:61]: If there are no more "Monitoring questions" updated/added, can we remove ALL "Monitoring-question"-features?

So also the related "exports", the storing of the answers, etc?

@RubenGeo Would you know?

(Might not be necessary to include in the PR, but could be added to the "collateral removals"-list.)

Comment (aberonni) [reply services/121-service/src/instance/instance.controller.ts:61]: @PeterSmallenbroek actually mentioned the same to me before realising you were already looking at this. I am going to address this in AB#26403

File: services/121-service/src/people-affected/person-affected-app-data.entity.ts
Comment (PeterSmallenbroek) [inline services/121-service/src/people-affected/person-affected-app-data.entity.ts:1]: Everything related to 'personaffectedappdata' can go now.
This was only used in the PA-app. We can remove:
- DB table & it's contents
- Entity
- Other mentions in code

**Review by PeterSmallenbroek — CHANGES_REQUESTED** [review]
Not sure where to put this elsewhere so I'll do it here.
get-user-auth-info-request.ts can also be removed

**Comment by aberonni** [comment]
@PeterSmallenbroek thanks for the review 🙏 

I prefer to merge this into main as-is, and then do the additional removals in a separate task, branch, and PR. That way we keep the changes as small as possible. Do you think that is an idea? Or is there a (bug/production) risk of merging the PR as-is?

**Comment by PeterSmallenbroek** [comment]
@aberonni  Okay, sure. I've created a task for the cleanup of PersonAffectedAppData

**Review by PeterSmallenbroek — APPROVED** [review]

**Pull request merged**